addons/hr_skills/static/src/fields/resume_one2many/resume_one2many.xml

```diff
@@ -40,7 +40,7 @@
             <td class="o_data_cell pt-0" t-on-click="(ev) => this.onCellClicked(record, null, ev)">
                 <div class="o_resume_line" t-att-data-id="id">
                     <small class="o_resume_line_dates fw-bold">
-                        <t t-esc="formatDate(data.date_start)"/> - <t t-if="data.date_end" t-esc="formatDate(data.date_end)"/><t t-else="">Current</t>
+                        <t t-esc="formatDate(data.date_start)"/> - <t t-if="data.date_end and formatDate(data.date_end) != formatDate(data.date_start)" t-esc="formatDate(data.date_end)"/><t t-else="">Current</t>
```

Comment:
Do we have to compare the formatted values? Seems strange.

Reply:
Well, it still looks strange, but hopefully we can rework that in the master PR so let's go with this solution in the mean time.